packages/ui/src/components/gas/GasPricesInfo.tsx

```diff
@@ -119,6 +120,8 @@ const GasPricesInfo: FC = () => {
         localeInfo,
         networkNativeCurrency,
         isNetworkChanging,
+        isRatesChangingAfterNetworkChange,
```

Comment:
Is isRatesChangingAfterNetworkChange needed here?

Reply:
It is not. Removed! 😁